openApiDocs/v1.0/Groups.yml

```diff
@@ -301,15 +296,11 @@ paths:
               description: $search and $count queries require the client to set the ConsistencyLevel HTTP header to 'eventual'.
               value: eventual
         - $ref: '#/components/parameters/top'
-        - $ref: '#/components/parameters/skip'
         - $ref: '#/components/parameters/search'
-        - $ref: '#/components/parameters/filter'
-        - $ref: '#/components/parameters/count'
```

Comment:
This doesn't look light. `/groups` definitely supports count and filter.

Reply:
This change will result in a breaking change. I've opened an issue under DevX API to track this https://github.com/microsoftgraph/microsoft-graph-devx-api/issues/611.